less/_grid.less

```diff
@@ -76,9 +76,13 @@
 
 // other CSS for the container of the new columns
 .grid {
+	@grid-sides: 15px;
 	.clearfix;
 
-	width: 100%;
+	padding: 0 25px 40px 25px;
```

Comment:
Should the grid itself have the bottom padding, or should that be something that requires explicit overrides?